staff/aleix-caminal/cv/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+    <head>
+        <meta charset="utf-8">
+        <title>Curriculum Vitae</title>
+    </head>
+    <body>
+        <div class="container">
```

Comment:
better use a more semantic tag instead of div

Reply:
there's no such tag as a container, moreover, div **is** the container tag

Reply:
changed it for a main anyways

Reply:
Hello guys, I was reading your comments and actually using `div` here in a semantic html document it's not correct. Maybe you could find a better way to name your container tag for example `<main>` or just create a section `<section>`.

Further information : [MDN Link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element)

Reply:
i've too much bootstrap in my head